atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schema.java

```diff
@@ -248,7 +249,7 @@ public void validate() {
             try {
                 entry.getValue().validate();
             } catch (Exception e) {
-                log.error("Failed to validate table {}.", entry.getKey(), e);
+                log.error("Failed to validate table {}.", UnsafeArg.of("tableName", entry.getKey()), e);
```

Comment:
These are table names from schemas, probably should be safe to log? Again, fine to leave as is